Weekly labels as clickable and cursor pointer

shivam178 opened the pull request:
### Description 💡 
This PR allows the cursor to be a pointer on hover & labels as clickable.

### Issue link 🔗 
https://github.com/sojinantony01/react-cron-generator/issues/74

### Screenshots 📷 


https://github.com/user-attachments/assets/c317ea24-fd8c-4540-978e-e000618433fb

shivam178 commented:
@sojinantony01 Kindly review.

sojinantony01 commented:
Hi @shivam178 
I am sorry for the delay, I mostly work on this in weekends
Thanks for the pull request, The changes looks good to me.
But as package is already using in multiple production applications we need to ensure consistent behaviour in all tabs, not only for weekly.
To make it consistent please do same for radio buttons as well. I hourly, daily, and monthly tabs.

shivam178 commented:
Updated UI:

https://github.com/user-attachments/assets/083fe662-4fee-4b6b-b744-5cd5ac7bf158


A few more things I've updated:
- Typo `coma` -> `comma`
- In the hourly & daily section, inputs were disabled if not selected, whereas in monthly that is not the case. Fixed with making them not disabled when not selected. Only a pro with this was if the user clicks directly on input, if enabled, it will select that radio button.
- Also in the monthly & daily section, updated the names of the input that are better and that are more suitable; there were duplicate names, which is not recommended.

sojinantony01 reviewed (commented):
Perfect, Thanks @shivam178 
I found an issue while testing. The number input boxes are not working as expected, I think you need to stopMousePropagation, prevent default of Mouse click events.

https://github.com/user-attachments/assets/80143889-9279-45e4-b252-60a1058e2bcc

shivam178 commented:
Fixed the issue:

https://github.com/user-attachments/assets/b6e74026-4d77-4bf8-b6f8-f666c9a913eb

Thanks for your careful review and your advice regarding Cypress tests.
I truly appreciate your efforts here, and I love your quick responsiveness.

sojinantony01 commented on src/lib/cron-builder.css:
```diff
@@ -125,3 +125,6 @@
 .cron_builder {
     font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,"Noto Sans",sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
 }
+.cursor_pointer {
```
Can we move this inside `.cron_builder` class? 
I don't want this to be affected in parent project classes.

sojinantony01 replied:
I havnt completed my review/test. 
Please wait for me to complete my tests, so that you can fix all changes at once 😄.

sojinantony01 replied:
The pointer is not working for me 🤔. (using Mac M1, Chrome)
I will test same again in this evening!. 
 
https://github.com/user-attachments/assets/af4b730e-dd5d-4f57-a6e8-0137dcbdfd60

shivam178 replied:
Hey @sojinantony01 Let me know if there's the pointer issue; I tried it in multiple browsers, and it's working well for me. I'm using M2 & Chrome/Safari.
> Can we move this inside .cron_builder class?
> I don't want this to be affected in parent project classes.
Sure, I can add the same once testing/reviewing is finalised.

sojinantony01 replied:
Hi, Sorry for the delay. I have finished my tests
The mouse pointer is working after a restart!
Not sure why it was not working before 😄. 

I am ready to approve the PR once the class pointer is moved inside cron_builder
@shivam178

shivam178 replied:
Done!

sojinantony01 reviewed (approved):
LGTM 👍  
Thanks @shivam178
